src/evm_transition_tool/evmone.py

```diff
@@ -110,63 +119,52 @@ def evaluate(
         )
 
         if debug_output_path:
+            shutil.copytree(temp_dir.name, debug_output_path)
+            t8n_output_base_dir = os.path.join(debug_output_path, "t8n.sh.out")
+            t8n_call = " ".join(args)
+            for file_path in input_paths.values():  # update input paths
+                t8n_call = t8n_call.replace(
+                    os.path.dirname(file_path), os.path.join(debug_output_path, "input")
+                )
+            t8n_call = t8n_call.replace(  # use a new output path for basedir and outputs
+                temp_dir.name,
+                t8n_output_base_dir,
+            )
+            t8n_script = textwrap.dedent(
```

Comment:
Since there is no output for evmone when running the `t8n.sh` script, we could echo the contents written to `$output_directory` - something like:
```python
t8n_script = textwrap.dedent(
    f"""\
    #!/bin/bash
    output_directory={os.path.join(t8n_output_base_dir, "output")}
    rm -rf $output_directory
    mkdir -p $output_directory
    {t8n_call}
    for file in $output_directory/*; do
    echo \-e "\\n>>> $(basename $file) <<<"
    cat $file
    done
    """
)
```

Reply:
We could do, but I'd leave as is for now; rodiazet already seems happy with it, at least with the addition of his changes in https://github.com/ethereum/execution-spec-tests/pull/273